### PR TITLE
[NHUB-547] Upgrade NewsAPI Atom feed to async

### PR DESCRIPTION
**Pull request by eos87**
### Purpose
To update `news_api.atom` and `news_api.assets` to use async resources as much as possible. `news_api.item` is still pending and should be addressed in a future PR once `formatters` are migrated to async. 

### What has changed
- Added two simple modules one for atom another for assets
- Adjusted some references, removed old apps from settings 

Resolves: [NHUB-547]

[NHUB-547]: https://sofab.atlassian.net/browse/NHUB-547?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: newsroom/news_api/news/assets/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from superdesk.core.module import SuperdeskAsyncApp, Module
+from .views import assets_endpoints
+
+
+def init_module(app: SuperdeskAsyncApp):
+    app.wsgi.register_endpoint(assets_endpoints)
```
Comment (MarkLark86) [inline newsroom/news_api/news/assets/__init__.py]: Endpoints can also be registered directly on the Module instance. Would that not work here?

File: newsroom/news_api/news/atom/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from superdesk.core.module import SuperdeskAsyncApp, Module
+from .views import atom_endpoints
+
+
+def init_module(app: SuperdeskAsyncApp):
+    app.wsgi.register_endpoint(atom_endpoints)
```
Comment (MarkLark86) [inline newsroom/news_api/news/atom/__init__.py]: Same here, regarding registering on Module instance instead

**Review by MarkLark86 — APPROVED** [review]
Looks good. Just a couple questions/improvements (as to not require init_app function)

File: newsroom/news_api/news/assets/__init__.py (hunk 1 above)
Comment (eos87) [reply newsroom/news_api/news/assets/__init__.py]: yeah, this is a leftover from when I was testing the registering of the resources in mongo and elastic directly. I will adjust it, thanks for spotting it.

**Pull request merged**